scripts/vercel_build.py

```
@@ -0,0 +1,40 @@
+    public_dir = Path("public")
+    public_dir.mkdir(exist_ok=True)
+    (public_dir / "placeholder.html").write_text("", encoding="utf-8")
+
+    compiler = shutil.which("g++")
+    if not compiler:
+        print("No g++, using Python engine")
+        return
+
+    output_path = Path("game/engine/main")
+    try:
+        subprocess.check_call(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)